etcdctl/ctlv3/command/member_command.go

```diff
@@ -111,8 +112,34 @@ func memberAddCommandFunc(cmd *cobra.Command, args []string) {
 	if err != nil {
 		ExitWithError(ExitError, err)
 	}
+	newID := resp.Member.ID
 
 	display.MemberAdd(*resp)
+
+	ctx, cancel = commandCtx(cmd)
+	listResp, err := mustClientFromCmd(cmd).MemberList(ctx)
+	cancel()
+
+	if err != nil {
+		ExitWithError(ExitError, err)
+	}
+
+	conf := []string{}
+	for _, memb := range listResp.Members {
+		for _, u := range memb.PeerURLs {
+			n := memb.Name
+			if memb.ID == newID {
+				n = newMemberName
+			}
+			conf = append(conf, fmt.Sprintf("%s=%s", n, u))
+		}
+	}
+
+	fmt.Print("\n")
+	fmt.Printf("ETCD_NAME=%q\n", newMemberName)
+	fmt.Printf("ETCD_INITIAL_CLUSTER=%q\n", strings.Join(conf, ","))
+	fmt.Printf("ETCD_INITIAL_CLUSTER_STATE=\"existing\"\n")
```

Comment:
could you the extra output from lines 127-141 into `display.MemberAdd(*resp)` from line 117? it will be cleaner that way.